specs/commandline/blob.md

```diff
@@ -0,0 +1,381 @@
+# notation policy
+
+## Description
+
+Use `notation blob` command to sign, verify, and inspect signatures associated with arbitrary blobs. Notation can sign and verify any arbitrary bag of bits like zip files, documents, executables, etc. When a user signs a blob, `notation` produces a detached signature, which the user can transport/distribute in any medium that the user prefers along with the original blob. On the verification side, Notation can verify the blob's signature and assert that the blob has not been tampered with during its transmission. For more details, see [trust policy specification and examples](https://github.com/notaryproject/specifications/blob/main/specs/signing-and-verification-workflow.md#blob-signing-workflow).
+
+## Outline
+
+### notation blob command
+
+```text
+Sign, inspect, and verify signatures associated with arbitrary blobs.
+
+Usage:
+  notation blob [command]
+
+Available Commands:
+  inspect   inspect a signature associated with a blob
+  sign      produce a detached signature for a given blob
+  verify    verify a signature associated with a blob
+
+Flags:
+  -h, --help   help for blob
+```
+
+### notation blob sign
+
+```text
+Produce a detached signature for a given blob. Signature file will be written to the currently working directory with blob file name + ".sig" + signature format as the file extension. For example, signature file name for "myBlob.bin" will be "myBlob.bin.sig.jws" for JWS signature format or "myBlob.bin.sig.cose" for COSE signature format.
+
+Usage:
+  notation blob sign [flags] <blob_path>
+
+Flags:
+  -sd,  --signature-directory path  optional path where the detached signature needs to be placed (default: currently working directory) 
```

Comment:
To alignment with flags below
```suggestion
  -sd, --signature-directory path  optional path where the detached signature needs to be placed (default: currently working directory) 
```

Reply:
I think the utility which we use to generate cli only allows for single character short flags.

Reply:
From UX perspective, `sd` makes sense. If the library doesn't support, we can revisit this.